From a78986634d20dc86e180a94faeec08b99b303327 Mon Sep 17 00:00:00 2001
From: Joshua Primero <jprimero155@gmail.com>
Date: Fri, 24 May 2019 11:34:16 +0000
Subject: [PATCH 2/7] msm8974-common: Nuke AdvancedDisplay

- We have livedisplay hal now so we can stop building advanced display.

Change-Id: I4aeaf71a29016b9f808777afed77a055d8b570b5
Signed-off-by: Joshua Primero <jprimero155@gmail.com>
---
 msm8974.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/msm8974.mk b/msm8974.mk
index 3bda121..b5b94d7 100644
--- a/msm8974.mk
+++ b/msm8974.mk
@@ -60,10 +60,6 @@ PRODUCT_PROPERTY_OVERRIDES += \
 PRODUCT_PROPERTY_OVERRIDES += \
     persist.timed.enable=true
 
-# Advanced Display
-PRODUCT_PACKAGES += \
-    AdvancedDisplay
-
 # Audio
 PRODUCT_PACKAGES += \
     android.hardware.audio@2.0-impl \
-- 
2.17.1

